docs/patterns/logo-wall/index.md

```diff
@@ -34,7 +34,7 @@ A logo wall is a visual arrangement of logos representing various brands, compan
   </div>
   <div>
     <h3>Borderless</h3>
-    <uxdot-example>
+    <uxdot-example width-adjustment="1010px"s>
```

Comment:
For this image, the intrinsic size is listed as `1113x110`. Needs updated for `<uxdot-example>` and the `<img>` tag.